pulp_container/app/serializers.py

```diff
@@ -627,6 +633,7 @@ def validate(self, data):
                 )
                 try:
                     artifact = artifactfield.run_validation(data=url)
```

Comment:
i can't grasp entirely when these artifacts are being saved since i don't see any further save call. I assume it is the `artifactfield.run_validation(data=url)` that creates and saves the artifacts? in that case we probably don't need that touch later.

Reply:
disregard, i have read the helptext for the `artifacts`